open-metadata-implementation/admin-services/admin-services-client/pom.xml

```diff
@@ -32,6 +32,10 @@
             <groupId>org.odpi.egeria</groupId>
             <artifactId>ffdc-services</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>ui-admin-services-api</artifactId>
```

Comment:
should not be dependent on the UI

Reply:
fixed

Reply:
This is not fixed?? May also relate to issue tracked in https://github.com/odpi/egeria/issues/2268. Will track followup there